target/linux/lantiq/files/arch/mips/boot/dts/lantiq/ar10_tplink_archer-d2.dts

```diff
@@ -0,0 +1,268 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "ar10.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mips/lantiq_rcu_gphy.h>
+
+/ {
+ 	compatible = "tplink,archer-d2", "lantiq,xway", "lantiq,ar10";
+ 	model = "TP-Link Archer D2 v1";
+
+ 	memory@0 {
+ 		device_type = "memory";
+ 		reg = <0x0 0x4000000>;
+ 	};
+
+	aliases {
+                led-boot = &led_power;
+                led-failsafe = &led_power;
+                led-running = &led_power;
+                led-upgrade = &led_power;
+
+                led-dsl = &led_dsl;
+                led-internet = &led_internet;
+                led-wifi = &led_wlan;
+        };
```

Comment:
Use tabs instead spaces. You can use checkpatch script to find similar minor mistakes.